core/dbt/task/run.py

```diff
@@ -887,77 +888,82 @@ def safe_run_hooks(
         failed = False
         num_hooks = len(ordered_hooks)
 
-        for idx, hook in enumerate(ordered_hooks, 1):
-            with log_contextvars(node_info=hook.node_info):
-                hook.index = idx
-                hook_name = f"{hook.package_name}.{hook_type}.{hook.index - 1}"
-                execution_time = 0.0
-                timing: List[TimingInfo] = []
-                failures = 1
-
-                if not failed:
-                    with collect_timing_info("compile", timing.append):
-                        sql = self.get_hook_sql(
-                            adapter, hook, hook.index, num_hooks, extra_context
+        if num_hooks == 0:
+            return status
+
+        tracer = trace.get_tracer("dbt-runner")
+        with tracer.start_as_current_span(hook_type, context=context.get_current()) as _:
```

Comment:
Isn't the default value `context.get_current()`?
```suggestion
        with tracer.start_as_current_span(hook_type) as _:
```